reference/docs-conceptual/SecretManagement/how-to/manage-secretstore.md

```diff
@@ -0,0 +1,157 @@
+encrypt file contents. The **SecretStore** module has several configuration options. In the default
+configuration, a password is required to store and access secrets, and provides the strongest
+protection. **SecretStore** also supports the storage of metadata about secrets.
+
+## Registering a new vault
+
+Before you can create new secret you must register a vault. The **Name** parameter is a friendly
+name and can be anything you choose.
+
+```powershell
+Register-SecretVault -Name SecretStore -ModuleName Microsoft.PowerShell.SecretStore -DefaultVault
+```
+
+> [!IMPORTANT]
+> The **SecretManagement** module allows an extension vault to be registered multiple times, because
+> it's possible that an extension vault may support different contexts via the registration
+> **VaultParameters**. However, the **SecretStore** vault currently always operates in the logged on
+> user scope. Registering multiple **SecretStore** vaults with different names just results in
+> duplication of the same store.
+
+Use the following command to see a list of registered vaults:
+
+```powershell
+Get-SecretVault
+
+Name        ModuleName                       IsDefaultVault
+----        ----------                       --------------
+Edge        SecretManagement.Chromium        False
+SecretStore Microsoft.PowerShell.SecretStore True
+```
+
+In this example there are two vaults registered using different extension modules.
+
+## Configuring a vault
+
+Use the `Get-SecretStoreConfiguration` cmdlet to see a vault's configuration.
+
+```powershell
+Get-SecretStoreConfiguration
+
+      Scope Authentication PasswordTimeout Interaction
+      ----- -------------- --------------- -----------
+CurrentUser       Password             900      Prompt
+```
+
+**SecretStore** vaults have the following configuration settings:
+
+- **Authentication** - `Password` (default) or `None`
+- **PasswordTimeout** - `900` seconds (default)
+- **Interaction** - `Prompt` (default) or `None`
+
+The default configuration requires a password, sets the session password timeout to 15 minutes, and
+prompts the user for a password to unlock the vault.
+
+For non-interactive automation scenarios, the **Interaction** can be configured as `None` to
+suppress user prompting. If a password is required, vault commands return
+**Microsoft.PowerShell.SecretStore.PasswordRequiredException** exception if there is no valid
+session password. The `Unlock-SecretStore` cmdlet can be used to provide the password for the
+current PowerShell session. The password remains valid until the password timeout expires.
+
+Vault configuration and data are stored in separate files. The file location depends on the platform
+operating system.
+
+- For Windows platforms the location is: `$env:LOCALAPPDATA\Microsoft\PowerShell\secretmanagement\localstore\`
+- For Non-Windows platforms the location is: `$HOME/.secretmanagement/localstore/`
```

Comment:
Are the file names deterministic? What are they?

Is the last segment of the path (`localstore`) deterministic to this implementation, or in general to vault type?